### ISSUE TITLE
Improve flood segmentation model 

### ISSUE THREAD

**Issue by melisandeteng** [cc-ai/kdb#109]
Improve the [current flood segmentation model](https://github.com/cc-ai/deeplab-pytorch) , see #62 

We report below a comparison of a first series of flood segmentation models using mIoU on our hand labeled dataset of 209 images from Keymakr. 

**DeepLab v2 with a ResNet-101 backbone (current model)**:
	- trained on COCO-Stuff 164K dataset
	- labels water-other, river, sand, sea were merged into the flood category
	mIoU ~ 79 % 

**DeepLab v2 with a ResNet-101 backbone + CRF postprocessing**:
	mIoU ~ 80 % 
	Extremely slow  (> 10s / image)

**[Sazara et. al ](https://arxiv.org/ftp/arxiv/papers/1909/1909.00125.pdf) - pretrained VGG-16 + CRF postprocessing**: 
	fine tuned on 202 hand-labeled images
	mIoU ~ 77% 
	slow : 7s/ image (and images had to be resized to have a max size of 600*600 pixels)

**Comment by melisandeteng** [comment]
Note that the masks for the flooded images will only need to be computed once. 
Next steps include : 
- Fine tune Deeplabv2 model with the flood hand-labeled images
- Qualitative comparison of the models, on our Keymakr test set but also images from other sources
- Report of the failure cases
- Investigate other segmentation models - Deeplabv3, HRNet

**Comment by vict0rsch** [comment]
How's that going @melisandeteng ?
